src/get-started/install.md

```diff
@@ -80,10 +80,12 @@ The Rust toolchain includes the Rust compiler (`rustc`) and several utilities,
 of which the most important one is [Cargo][cargo], the Rust package manager.
 
 !!! note
-    Exonum is tested against and is guaranteed to be compatible with
+    Exonum is guaranteed to be compatible with
     the latest stable Rust toolchain, which can be obtained as specified above.
-    Other recent toolchains (starting from approximately `stable-1.20`) may work
-    too. Older toolchains (for example, those that come with older Linux
+    Other recent toolchains may work too; see
+    [the Exonum main repository readme][readme] to find out
+    a precise range of supported Rust versions.
+    Older toolchains (for example, those that come with older Linux
```

Comment:
Note related to this PR, but "older Linux distribution" does not seem entirely correct to me: an up-to-date linux distribution may include older Rust. I suggest just dropping this word.